man/fi_info.1.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: fi_info(1)
+tagline: Libfabric Programmer's Manual
+---
+{% include JB/setup %}
+
+
+# NAME
+
+fi_info  \- Simple utility to query for fabric interfaces
+
+
+# SYNOPSYS
+```
+ fi_info [OPTIONS]
+```
+
+# DESCRIPTION
+
+The fi_info utility can be used to query for available fabric interfaces. The
+utility supports filtering based on a number of options such as endpoint type,
+provider name, or supported modes. Additionally, fi_info can also be used to
+discover the environment variables that can be used to tune provider specific
+parameters.
```

Comment:
> Since the filters are optional, should we clarify that it returns all available fabric interfaces for all providers and endpoint types when no filters are specified?

I can add that.